src/Element/MediaMpxIframeWrapper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\media_mpx\Element;
+
+use Drupal\Core\Render\Element\RenderElement;
+use Drupal\Core\Template\Attribute;
+
+/**
+ * Providers an element design for wrapping media iframes.
+ *
+ * @RenderElement("media_mpx_iframe_wrapper")
+ */
+class MediaMpxIframeWrapper extends RenderElement {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getInfo() {
+    return [
+      '#theme' => 'media_mpx_iframe_wrapper',
+      '#url' => '',
+      '#attributes' => [],
+      '#pre_render' => [
+        [static::class, 'preRenderInlineFrameEmbed'],
+      ],
+    ];
+  }
+
+  /**
+   * Transform the render element structure into a renderable one.
+   *
+   * @param array $element
+   *   An element array before being processed.
+   *
+   * @return array
+   *   The processed and renderable element.
+   */
+  public static function preRenderInlineFrameEmbed(array $element) {
+    if (is_array($element['#attributes'])) {
+      $element['#attributes'] = new Attribute($element['#attributes']);
```

Comment:
Ooo interesting, I thought core handled `#attributes` for you automatically this way.

Reply:
Hah, you got this from my code I wrote back in April, so clearly I ran into this already.